### DISCUSSION TITLE
More eu-friendly meeting times

### DISCUSSION THREAD

**Discussion by niklasmtj** [open-gitops/project#158] [General]
As discussed at KubeCon and in the meetings, there is a desire for a more "EU-friendly" time for the bimonthly meeting. There have been a few ideas from folks in the group in the past. So this discussion should be the starting point for more defined plans on how we would like to implement this.

1. Move the meeting to 4pm GMT. (9am PST, 12pm EST, 6pm CEST) - time tbd
2. Set up a second meeting focused on EU time zones.
3. 3rd option is have the 1st Wed meeting be in a EU friendly timezone and the 3rd Wed keep the current time. - proposed by @christianh814 (putting this here to give a better overview of the options)

What do you think about that?

**Comment by christianh814** [comment]
Possible 3rd option is have  the 1st Wed meeting be in a EU friendly timezone and the 3rd Wed keep the current time.

**Comment by nikimanoledaki** [comment]
+1 for the 1st option but open to all options

cc @makkes @nishadmehendale who have shown interest in this previously [here](https://github.com/open-gitops/project/discussions/103) 😊

**Comment by tsahiduek** [comment]
IMO it depends on what is the difference between option 1 and 3. 
If option 3 means it will be at 4 CEST, then it will be better because it won't be too late for those in the middle-east region as well

**Comment by makkes** [comment]
We are running pretty well with option 3 in the Flux project meeting. However, a common meeting time that fits both continents would be preferable. 6pm CEST is too late for me personally so I'd vote for 5pm CEST (3pm UTC, 8am PT) but I understand this is pretty early for west coast folks.

**Reply by nikimanoledaki** [reply]
It's great that there is a precedent for option 3 in the Flux project meeting that we can learn from.

My only concern (echoed by @williamcaban in yesterday's meeting) is to avoid overlapping with the Environmental Sustainability TAG meeting if possible. Currently, this runs every 1st and 3d Wednesday at 3pm UTC / 5pm CEST / 8am PDT.

**Reply by wwentland** [reply]
> 6pm CEST is too late for me personally

I'm sure that this is a problem for many people in Europe. 3pm UTC / 5pm CEST / 8am PDT is indeed the best compromise that can be found here for attendees in central Europe and the US West Coast.

**Comment by wwentland** [comment]
If the aim is to have a meeting that is genuinely EU/EMEA friendly, than it would be best to schedule it during working hours.

My personal preference is to have a rotating meeting schedule so that everyone can attend meetings within core working hours. To be fair to people around the globe, maybe rotating through the following slots would be an idea (option 4)?

- 09:00 America/Los_Angeles - Core working hours in the Americas, possible for Europe
- 09:00 Europe/London - Core Europe/Middle East working hours, possible for East Asia/Oceania
- 09:00 Asia/Shanghai - Core Asia and Oceania working hours, possible for the Americas

(cf. https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
